src/tests/system/tests/test_passkey.py

```diff
@@ -544,3 +544,40 @@ def test_passkey__su_fips_fido_key(client: Client, provider: GenericProvider, mo
         ioctl=f"{moduledatadir}/umockdev.ioctl",
         script=f"{testdatadir}/umockdev.script.{suffix}",
     )
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.builtwith(client="passkey", ipa="passkey")
+@pytest.mark.require.with_args(passkey_requires_root)
+def test_passkey__check_tgt(client: Client, ipa: IPA, moduledatadir: str, testdatadir: str):
+    """
+    :title: Check the TGT of user after authentication.
+    :setup:
+        1. Add a user with --user-auth-type=passkey in the server with passkey mapping.
+        2. Setup SSSD client with FIDO and umockdev, start SSSD service.
+    :steps:
+        1. Check authentication of the user
+        2. Check TGT after authenticates.
+    :expectedresults:
+        1. User authenticates successfully.
+        2. Gets the TGT.
+    :customerscenario: False
+    """
+
+    with open(f"{testdatadir}/passkey-mapping.ipa") as f:
+        ipa.user("user1").add(user_auth_type="passkey").passkey_add(f.read().strip())
+
+    client.sssd.start()
+
+    rc, _, output, _ = client.auth.su.passkey_with_output(
+        username="user1",
+        pin=123456,
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.ipa",
+        command="klist",
+    )
+
+    assert rc == 0
+    assert "Ticket cache" in output
```

Comment:
Add a meaningful error message in case the assert fails. You have a similar example in https://github.com/SSSD/sssd/pull/7393/files#diff-d6f662469d57a0e4deb526c1d7e86dd9402ebd45116869053828053506d0007aL509

Reply:
updated.